eldritch/mythos.py

```diff
@@ -456,6 +605,44 @@ def get_trigger(self, event, state):
     return super().get_trigger(event, state)
 
 
+class Mythos28(Headline):
+  def __init__(self):
+    super().__init__("Mythos28", "Graveyard", "Isle", {"hex"}, {"slash", "triangle", "star"})
+
+  def create_event(self, state):
+    first_player = state.characters[state.first_player]
+    seq = super().create_event(state)
+    check = events.Check(first_player, "luck", -1)
+    pass_fail = events.PassFail(first_player, check, events.Nothing(), events.Curse(first_player))
+    seq.events.extend([check, pass_fail])
+    return seq
+
+
+class Mythos29(Environment):
+  def __init__(self):
+    super().__init__("Mythos30", "Society", "Lodge", {"plus"}, {"moon"}, "weather")
+
+  def get_override(self, thing, attribute):
+    if isinstance(thing, monsters.Monster) and attribute == "fast":
+      return "normal"
+    return None
+
+  def get_interrupt(self, event, state):
+    if isinstance(event, events.Movement):
+      return events.ChangeMovementPoints(event.character, -1)
```

Comment:
Note that because this is an interrupt, it will run before Movement does. But Movement is the event that actually grants movement points. So I think you want to interrupt CityMovement, not Movement. Please add a test.